velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -1617,6 +1617,33 @@ bool AggregationFuzzer::verifySortedAggregation(
              .planNode(),
          splits});
 
+    if (!groupingKeys.empty()) {
```

Comment:
This should be outside of the isTableScanSupported 'if'.

Reply:
Fixed